libpcsxcore/psxinterpreter.c

```diff
@@ -217,14 +213,12 @@ static u32 INT_ATTR fetchICache(psxRegisters *regs, u8 **memRLUT, u32 pc)
 
 		if (((entry->tag ^ pc) & 0xfffffff0) != 0 || pc < entry->tag)
 		{
-			const u8 *base = memRLUT[pc >> 16];
-			const u32 *code;
-			if (unlikely(base == INVALID_PTR)) {
+			const u32 *code = (u32 *)psxm(pc & ~0x3, 0);
```

Comment:
This should be ~0x0f since it's fetching a cacheline. Please test with "ICache emulation" enabled.

Reply:
I used `~0x3` because the original mask was `0xfffc`, but I can use `~0xf` yes.

Reply:
It wasn't:
```
code = (u32 *)(base + (pc & 0xfff0));
```